### PR TITLE
Tidy up tracing statements

### PR DESCRIPTION
**Pull request by thomaseizinger**

### CODE REVIEW

File: swap/src/asb/event_loop.rs [hunk 1]
```diff
@@ -176,13 +176,13 @@ where
                             let _ = self.handle_execution_setup_done(peer_id, swap_id, *state3).await;
                         }
                         SwarmEvent::Behaviour(OutEvent::SwapDeclined { peer, error }) => {
-                            tracing::warn!(%peer, "Ignoring spot price request because: {}", error);
+                            tracing::warn!(%peer, "Ignoring spot price request: {}", error);
```
Comment (da-kami) [inline swap/src/asb/event_loop.rs:179]:
```suggestion
                            tracing::warn!(%peer, "Ignoring spot price request: {:#}", error);
```

File: swap/src/asb/event_loop.rs [hunk 2]
```diff
@@ -350,11 +350,11 @@ where
         match self.db.insert_peer_id(swap_id, bob_peer_id).await {
             Ok(_) => {
                 if let Err(error) = self.swap_sender.send(swap).await {
-                    tracing::warn!(%swap_id, "Swap cannot be spawned: {}", error);
+                    tracing::warn!(%swap_id, "Failed to start swap: {}", error);
```
Comment (da-kami) [inline swap/src/asb/event_loop.rs:353]:
```suggestion
                    tracing::warn!(%swap_id, "Failed to start swap: {:#}", error);
```

File: swap/src/asb/event_loop.rs [hunk 3]
```diff
@@ -350,11 +350,11 @@ where
...
                 }
             }
             Err(error) => {
-                tracing::warn!(%swap_id, "Unable to save peer-id, swap cannot be spawned: {}", error);
+                tracing::warn!(%swap_id, "Unable to save peer-id in database: {}", error);
```
Comment (da-kami) [inline swap/src/asb/event_loop.rs:357]:
```suggestion
                tracing::warn!(%swap_id, "Unable to save peer-id in database: {:#}", error);
```

File: swap/src/network/tor_transport.rs [hunk 4]
```diff
@@ -42,7 +42,7 @@ impl Transport for TorDialOnlyTransport {
         }
 
         let dial_future = async move {
-            tracing::trace!("Connecting through Tor proxy to address: {}", addr);
+            tracing::trace!("Connecting through Tor proxy to address {}", addr);
```
Comment (da-kami) [inline swap/src/network/tor_transport.rs:45]: Make the `addr` a field?

File: swap/src/protocol/bob/swap.rs [hunk 5]
```diff
@@ -147,11 +147,13 @@ async fn next_state(
                     received_xmr = monero_wallet.watch_for_transfer(watch_request) => {
                         match received_xmr {
                             Ok(()) => BobState::XmrLocked(state.xmr_locked(monero_wallet_restore_blockheight)),
-                            Err(e) => {
-                                 tracing::warn!("Waiting for refund because insufficient Monero have been locked! {:#}", e);
-                                 tx_lock_status.wait_until_confirmed_with(state.cancel_timelock).await?;
+                            Err(monero::InsufficientFunds { expected, actual }) => {
+                                tracing::warn!("Insufficient Monero have been locked! Expected {} but got {}", expected, actual);
```
Comment (da-kami) [inline swap/src/protocol/bob/swap.rs:151]: Change variables as part of message to fields?

File: swap/src/protocol/bob/swap.rs [hunk 6]
```diff
@@ -147,11 +147,13 @@ async fn next_state(
...
+                                tracing::info!("Waiting for cancel timelock ({}) to expire", state.cancel_timelock);
```
Comment (da-kami) [inline swap/src/protocol/bob/swap.rs:152]: Change variables as part of message to fields?

File: swap/src/seed.rs [hunk 7]
```diff
@@ -64,7 +64,7 @@ impl Seed {
             return Self::from_file(&file_path);
         }
 
-        tracing::debug!("No seed file found, creating at: {}", file_path.display());
+        tracing::debug!("No seed file found, creating at {}", file_path.display());
```
Comment (da-kami) [inline swap/src/seed.rs:67]: Change variables as part of message to fields?

**Review by da-kami — APPROVED** [review]

File: swap/src/asb/event_loop.rs (hunk 1 above)
Comment (thomaseizinger) [reply swap/src/asb/event_loop.rs:179]: It is not an `anyhow::Error` so I don't think the alternate `Display` is going to do anything useful.

File: swap/src/asb/event_loop.rs (hunk 2 above)
Comment (thomaseizinger) [reply swap/src/asb/event_loop.rs:353]: It is not an `anyhow::Error` so I don't think the alternate `Display` is going to do anything useful.

File: swap/src/asb/event_loop.rs (hunk 3 above)
Comment (thomaseizinger) [reply swap/src/asb/event_loop.rs:357]: It is not an `anyhow::Error` so I don't think the alternate `Display` is going to do anything useful.

**Comment by thomaseizinger** [comment]
@da-kami Do you mind pushing a follow-up commit that works in the suggestions you made? That would be nice!

**Review by rishflab — APPROVED** [review]

**Pull request merged**